BP/common/culture/cultures/00_western_germanic.txt

```diff
@@ -0,0 +1,1193 @@
+			"dynn_Agilolfing"
+			{ "dynnp_von" "dynn_Argengau" }
+			"dynn_Ascaricing"
+			"dynn_Ascaring"
+			"dynn_Australdling"
+			"dynn_Baugulfson"
+			"dynn_Berengling"
+			{ "dynnp_de" "dynn_Betoricas" }
+			"dynn_Bouvinid"
+			"dynn_Charibertid"
+			"dynn_Charing"
+			"dynn_Chiemgaudson"
+			"dynn_Chloding"
+			"dynn_Chlothings"
+			"dynn_Cloving"
+			"dynn_Dodonid"
+			"dynn_Drogling"
+			"dynn_Emichid"
+			"dynn_Engelgerson"
+			"dynn_Erenolding"
+			"dynn_Galimani"
+			"dynn_Genobauding"
+			"dynn_Gunberts"
+			"dynn_Herluins"
+			"dynn_Hermenelds"
+			"dynn_Hetanid"
+			"dynn_Hildegars"
+			"dynn_Hildradson"
+			"dynn_Hucberting"
+			"dynn_Hugonid"
+			"dynn_Hugonid"
+			"dynn_Laidrati"
+			{ "dynnp_de" "dynn_Landen" }
+			"dynn_Launebodis"
+			"dynn_Leudoni"
+			"dynn_Luping"
+			"dynn_Mallobauding"
+			"dynn_Maracharing"
+			"dynn_Marcoming"
+			"dynn_Merogaising"
+			"dynn_Meroving"
+			"dynn_Morgannid"
+			"dynn_Nibelunging"
+			"dynn_Pharamonding"
+			"dynn_Ragnaching"
+			"dynn_Riccing"
+			"dynn_Riculfling"
+			"dynn_Rignoming"
+			"dynn_Ringling"
+			"dynn_Roding"
+			"dynn_Samosing"
+			"dynn_Sigobing"
+			"dynn_Sunning"
+			"dynn_Theodnid"
+			"dynn_Theuding"
+			{ "dynnp_de" "dynn_Transjuranie" }
+			"dynn_Vuodi"
+			"dynn_Walthars"
+			"dynn_Warins"
+			"dynn_Wegelenzi"
+			"dynn_Welenti"
+			"dynn_Widling"
+			"dynn_Wulfardson"
+		}
+
+		dynasty_names = {
+			"dynn_Adalberts"
+			"dynn_Adelards"
+			"dynn_Aegiding"
+			"dynn_Agilolfing"
+			{ "dynnp_von" "dynn_Argengau" }
+			"dynn_Ascaricing"
+			"dynn_Ascaring"
+			"dynn_Australdling"
+			"dynn_Baugulfson"
+			"dynn_Berengling"
+			{ "dynnp_de" "dynn_Betoricas" }
+			"dynn_Bouvinid"
+			"dynn_Charibertid"
+			"dynn_Charing"
+			"dynn_Chiemgaudson"
+			"dynn_Chloding"
+			"dynn_Chlothings"
+			"dynn_Cloving"
+			"dynn_Dodonid"
+			"dynn_Drogling"
+			"dynn_Emichid"
+			"dynn_Engelgerson"
+			"dynn_Erenolding"
+			"dynn_Galimani"
+			"dynn_Genobauding"
+			"dynn_Gunberts"
+			"dynn_Herluins"
+			"dynn_Hermenelds"
+			"dynn_Hetanid"
+			"dynn_Hildegars"
+			"dynn_Hildradson"
+			"dynn_Hucberting"
+			"dynn_Hugonid"
+			"dynn_Hugonid"
+			"dynn_Laidrati"
+			{ "dynnp_de" "dynn_Landen" }
+			"dynn_Launebodis"
+			"dynn_Leudoni"
+			"dynn_Luping"
+			"dynn_Mallobauding"
+			"dynn_Maracharing"
+			"dynn_Marcoming"
+			"dynn_Merogaising"
+			"dynn_Meroving"
+			"dynn_Morgannid"
+			"dynn_Nibelunging"
+			"dynn_Pharamonding"
+			"dynn_Ragnaching"
+			"dynn_Riccing"
+			"dynn_Riculfling"
+			"dynn_Rignoming"
+			"dynn_Ringling"
+			"dynn_Roding"
+			"dynn_Samosing"
+			"dynn_Sigobing"
+			"dynn_Sunning"
+			"dynn_Theodnid"
+			"dynn_Theuding"
+			{ "dynnp_de" "dynn_Transjuranie" }
+			"dynn_Vuodi"
+			"dynn_Walthars"
+			"dynn_Warins"
+			"dynn_Wegelenzi"
+			"dynn_Welenti"
+			"dynn_Widling"
+			"dynn_Wulfardson"
+		}
+		
+		male_names = {
+			Abrahil Abram Adalbald Adalbert Adalgari Adalgrim Aegidius Ageric Agilbert Agiulf Ailbert Albric Aldedramn Andica Ansovald Arcambald 
```

Comment:
This is our name list but like looking at Vanilla's they honestly seem to be very similar as well and vanilla's is slightly larger.

Through Git the different vanilla names are - Arnulf, Bertrand, Karloman, Chlothar, Karl, Lambert, Niebelung and Roland for men and Aldegonde, Anstrude, Austreberta, Balthild, Begga, Bertilla, Burgundofara, Cesaria, Eustadiola, Genovefa, Gertrude, Glodesind, Itta, Monegund, Rictrude, Rusticula, Sadalberga and Waltrude.

For men however the names Bertramn, Bertrann and Carloman are missing.

Reply:
Karloman can be added definitely, Bertramn kept as it is in the PR line 808, for Bertrann I don't know about the spelling change, would have to do some research. Bertrand to my knowledge is a change from "Bertramn", I think keeping the latter makes more sense. Cesaria from the women list I wouldn't add in 476 since that seems to be based upon religious Christian customs with influence from Latin. Only after conversion that name would work. The rest seems like it is fine.